dbfit-java/sqlserver/src/integration-test/resources/acceptancescripts-SqlServer.sql

```diff
@@ -82,6 +82,22 @@ SET ANSI_NULLS ON
 GO
 SET QUOTED_IDENTIFIER ON
 GO
+IF NOT EXISTS (SELECT * FROM sys.objects WHERE object_id = OBJECT_ID(N'[dbo].[Increment_P]') AND type in (N'P', N'PC'))
+BEGIN
+EXEC dbo.sp_executesql @statement = N'CREATE PROCEDURE [dbo].[Increment_P]
+	@counter INT OUTPUT
+AS
+BEGIN
+	set @counter = isnull(@counter, 1) + 1
```

Comment:
We have no defined standard style for SQL code but elsewhere we have the keyword `SET` and built-in functions in UPPER case.